btp/autoscaler/cpu_scaling_policy.json

```diff
@@ -0,0 +1,22 @@
+{
+  "instance_min_count": 1,
+  "instance_max_count": 2,
+  "scaling_rules": [
+    {
+      "metric_type": "cpuutil",
+      "breach_duration_secs": 500,
+      "threshold": 75,
+      "operator": ">",
+      "cool_dow_secs": 250,
+      "adjustment": "+1"
+    },
+    {
+      "metric_type": "cpuutil",
+      "breach_duration_secs": 500,
+      "threshold": 30,
```

Comment:
in case you defined `threshold >= 75` in first rule, you have to define the same value here (`threshold < 75`), otherwise it's quite difficult to recognize behaviour of scaling